src/PotentialTypes/BasisPotentials/SNAP/utilities/utl.jl

```diff
@@ -68,26 +67,23 @@ function neighborlist(A::AbstractSystem, snap::SNAPParams)
     for n = 1:length(X)
         n_element = findall(x->x==Symbol(A[n].atomic_symbol), elements)[1]
         neighbors = inrange(tree, X[n], sqrt(cutmax), true)
-        itemp = Int64[]
         jtemp = Int64[]
         Rtemp = Float64[]
         rtemp = SVector{3, Float64}[]
-        for m in 1:length(neighbors)
+        for (m, neighbor) in enumerate(neighbors)
             m_element = findall(x->x==Symbol(A[m].atomic_symbol), elements)[1]
```

Comment:
Replace `Symbol(A[m].atomic_symbol)` with `atomic_symbol(A, m)` -- depending on the concrete implementation of `A` the current version may be inefficient or not work entirely; the result of the function is already a symbol, so no need to convert it.